sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactListCriteriaBuilder.java

```diff
@@ -297,4 +304,39 @@ public Predicate buildContactFilter(
 
 		List<Expression<?>> forProperty(SortProperty sortProperty, Root<Contact> contact, ContactJoins joins);
 	}
+
+	public List<Selection<?>> getContactIndexSelectionsAll(Root<Contact> contact, ContactJoins joins) {
```

Comment:
Yes because it's in relation with the second constructor wich doesn't have some parameters in ContactIndexDto, getCommunity, get(Person.SYMPTOM_JOURNAL_STATUS), get(Contact.CHANGE_DATE) and get(Contact.EXTERNAL_ID)

Reply:
Could you then maybe restructure the constructors in a way that puts the additional variables to the end of it? You then don't need two full getContactIndexSelections methods, but only a method that appends the additional variables to the selections made by the existing getContactIndexSelections method